helm_deploy/hmpps-prisoner-to-nomis-update/templates/report-appointments-cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-report-appointments
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "5 0 * * THU"
+  suspend: false
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 600
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      # Tidy up all jobs after 4 days
+      ttlSecondsAfterFinished: 345600
+      template:
+        spec:
+          containers:
+            - name: report-appointments
```

Comment:
@mikehalmamoj  has added a preferred method of doing this with hmpps auth tokens - might be worth asking if that is still relevant since I think he is reworking cron jobs